src/components/Nav/Nav.js

```diff
@@ -33,9 +44,15 @@ const Nav = () => {
         </div>
       </div>
       <div className="categoryWrap">
-        <div className="categoryWrapLeft">
+        <div className="categoryWrapLeft" onMouseEnter={hoverCategory}>
```

Comment:
```suggestion
       <div
          className="categoryWrapLeft"
          onMouseEnter={() => hoverCategory(true)}
        >
```
위의 리뷰와 연결하자면 이런 방식이 되겠습니다.